src/instructlab/defaults.py

```diff
@@ -11,11 +11,11 @@
 CONFIG_VERSION = "1.0.0"
 
 # Model families understood by ilab
-MODEL_FAMILIES = {"merlinite", "mixtral"}
+MODEL_FAMILIES = {"granite", "granite_legacy", "mixtral"}
 
 # Map model names to their family
 MODEL_FAMILY_MAPPINGS = {
-    "granite": "merlinite",
+    "merlinite": "granite_legacy",
```

Comment:
What happens when the current version of things is superseded? Will it be renamed to `granite_legacy2`?

Reply:
updated this to not introduce a legacy family, but instead map all granite models t o"granite" and then differentiate on architecture